src/browser/modules/Frame/styled.jsx

```diff
@@ -87,7 +87,7 @@ export const StyledFrameMainSection = styled.div`
 `
 
 export const StyledFrameAside = styled.div`
-  flex: 1 0 25%;
+  flex: 0 0 25%;
```

Comment:
This change is scary. I haven't review it visually yet so it might be fine, but this is a major change. :)

Reply:
It might look scary, but the Aside (which I added in the major PR we merged a few weeks ago) is not supposed to grow.